src/index.js

```diff
@@ -97,8 +97,22 @@ class LumigoPlugin {
 		).toLowerCase();
 	}
 
+	get useServerlessEsbuild() {
```

Comment:
Will the new approach work also for versions before 1.30?

Reply:
Yes